stablehlo/tests/compat_downgrade.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: stablehlo-opt --stablehlo-legalize-to-versionedhlo --versionedhlo-to-version='target=minimum' %s | FileCheck %s
```

Comment:
Speaking of aliases, what's the use case for having the `minimum` alias? 

I can see how `current` is useful. When deserializing StableHLO programs, I'd want to run `vhlo-to-version=current` without thinking what exactly is the version I want, and then I'd want to `vhlo-legalize-to-stablehlo`.

What are the use cases where someone would want to indiscriminately downgrade to `minimum`, regardless of what version that actually is?

Reply:
Intention was for a user who wouldn't want to have to continue updating the minimum supported target version. I.e. producer is on 0.5.x and some consumers are running a slightly different versions depending on their update cycle 0.2.x, 0.6.x. Targeting minimum is seamless and safe. The alternative would be knowing exactly what the minimum consumer number is and targeting that, and then updating that number any time the minimum consumer number changes.

That said, it probably isn't that useful to target minimum. Especially if producers want to start using new features that aren't in the minimum supported version, then they will need to go to the above system where they target the minimum (or less than minimum) consumer version.

I agree we should remove this. Tests can be updated to `0.0.0`.